README.md

```diff
@@ -8,15 +8,14 @@ This repository is a [monorepo] containing the reference implementation of D3N a
 
 ## Table of Contents
 
-| Module                 | Description                                               |
-| ---------------------- | --------------------------------------------------------- |
-| [`chain`](chain)       | 🔗 D3N blockchain reference implementation                |
-| [`explorer`](explorer) | 🔍 Web interface to explore D3N network                   |
-| [`bridges`](bridges)   | 📡 Lite client bridges on other smart contract platforms  |
-| [`owasm`](owasm)       | 🔮 WebAssembly library for writing oracle scripts         |
-| [`scan`](scan)         |                                                           | 
-| [`spec`](spec)         | 📖 D3N research and specification knowledge base          |
-| [`studio`](studio)     | 🎬 In-browser IDE for testing and deploying owasm scripts |
+| Module               | Description                                               |
+| -------------------- | --------------------------------------------------------- |
+| [`chain`](chain)     | 🔗 D3N blockchain reference implementation                |
+| [`bridges`](bridges) | 📡 Lite client bridges on other smart contract platforms  |
+| [`owasm`](owasm)     | 🔮 WebAssembly library for writing oracle scripts         |
+| [`scan`](scan)       |                                                           |
```

Comment:
Can you add `🔍 Web interface to explore D3N network` to scan?

Reply:
Sure